server/index.js

```diff
@@ -1,12 +1,40 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const path = require('path');
 const db = require('../database/index.js');
+const helpers = require('./queryHelpers.js');
 
 const app = express();
 const port = 3002;
 
+// app.use(express.static('../public'));
+app.use(express.static(`${__dirname}/../client/dist/`));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
-app.use(express.static('../public'));
 
-app.listen(port, () => console.log(`Server listening on port ${port}`));
\ No newline at end of file
+
+app.get('/rooms/:homeid/reviews', (req, res) => {
+  const homeID = req.params.homeid;
+  const keyword = req.query.keyword || undefined;
+  // get data from database
+  if (keyword === undefined) {
+    helpers.getReviews(homeID, (results) => {
```

Comment:
Seems like if you get an error from `queryHelpers.js`, this request would just hang, and no response ever provided to client. I'm currently doing the same thing, so not sure what's the best way to handle it. would like to discuss.

Reply:
Thanks for the helpful discussion! I've added error handling in index.js and made sure errors are passed to the callbacks in the helper functions.